src/styles/blocks/page.scss

```diff
@@ -0,0 +1,6 @@
+.page {
+  &__stopwatch > * {
+    top: 50%;
+    left: 50%;
+  }
```

Comment:
Don't use selectors like this.
You can freely position BEM-element just using classes.